src/view/com/util/Toast.web.tsx

```diff
@@ -78,12 +143,12 @@ const styles = StyleSheet.create({
     bottom: 20,
     // @ts-ignore web only
     width: 'calc(100% - 40px)',
-    maxWidth: 350,
+    maxWidth: 380,
```

Comment:
made the width of the toast bigger so that its not as compressed when there is a full sentence or a few sentences in the toast

Reply:
In the future leave this as a comment in the code as

```
// NOTE(APiligrim)
// made the width of the toast bigger so that its not as compressed when there is a full sentence or a few sentences in the toast
```